server/data/data-library/src/main/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTable.java

```diff
@@ -119,14 +119,14 @@ public Mappings getMappings(String user, Domain domain, int mappingLimit) throws
                 MappingsImpl.Builder mappings = MappingsImpl.builder();
 
                 for (String target : targetMappings.asStrings()) {
-                    if (target.startsWith(RecipientRewriteTable.REGEX_PREFIX)) {
+                    if (target.startsWith(Type.Regex.asPrefix())) {
```

Comment:
`Type.detectType(target)` ?

Reply:
Done in https://github.com/linagora/james-project/pull/1350